src/containers/Main.js

```diff
@@ -30,22 +44,28 @@ const Main = () => {
   return (
     <div className={isDark ? "dark-mode" : null}>
       <StyleProvider value={{isDark: isDark, changeTheme: changeTheme}}>
-        <Header />
-        <Greeting />
-        <Skills />
-        <StackProgress />
-        <Education />
-        <WorkExperience />
-        <Projects />
-        <StartupProject />
-        <Achievement />
-        <Blogs />
-        <Talks />
-        <Twitter />
-        <Podcast />
-        <Profile />
-        <Footer />
-        <Top />
+        {isShowingSplashAnimation && splashScreen.enabled ? (
+          <SplashScreen />
+        ) : (
+          <>
+            <Header />
+            <Greeting />
+            <Skills />
+            <StackProgress />
+            <Education />
+            <WorkExperience />
+            <Projects />
+            <StartupProject />
+            <Achievement />
+            <Blogs />
+            <Talks />
+            <Twitter />
+            <Podcast />
+            <Profile />
+            <Footer />
+            <Top />
```

Comment:
Unrelated to your changes but can we rename this component to ScrollToTopButton or something? 
Will make the code easier to understand 😅

Reply:
Since this was a default import, I just changed the name in this file and did not modify the actual component code. Do let me know if it was to be done differently.